blocks/Select/Select.js

```diff
@@ -0,0 +1,53 @@
+import { Block, BaseComponent } from '../../abstract/Block.js';
+
+export class Option extends BaseComponent {
+  initCallback() {
+    this.outerHTML = /*html*/ `<option value="${this.$.value}">${this.$.text}</option>`;
+  }
+}
+Option.reg('lr-option');
```

Comment:
let's not register components in the module scope, that leads to troubles on SSR

Reply:
I've removed repeat and repeat item, that was wrong approach more like experiment...